index.js

```diff
@@ -16,7 +16,12 @@ const give = (data) =>
 
 const init = async () => {
   const [path, filetype] = process.argv.slice(2);
-  const lang = filetype[0] === "t" ? Lang.TYPESCRIPT : Lang.JAVASCRIPT;
+  const lang = (() => {
```

Comment:
Rather than wrap this up in a function with if statements, I think a map is meant to solve this problem. Then we can get the lang like below:

```javascript
const lang = filetypes.get(filetype[0]);
```